src/mono/wasm/Makefile

```diff
@@ -91,6 +92,28 @@ build-interp-$(1): $(BUILDS_BIN_DIR)/$(1)/dotnet.js
 
 endef
 
+# Needed by AOT
+
+SRC_FILES=\
+	runtime/driver.c \
+	runtime/corebindings.c \
+	runtime/binding_support.js \
+	runtime/dotnet_support.js \
+	runtime/library_mono.js
```

Comment:
these .js files will need to be added to https://github.com/dotnet/runtime/blob/master/eng/SignCheckExclusionsFile.txt#L15 to workaround the signing issue.